.github/workflows/fuzz-nightly.yml

```diff
@@ -49,14 +43,14 @@ jobs:
         with:
           name: crashers
           path: test/fuzz/**/crashers
-          retention-days: 1
+          retention-days: 3
 
       - name: Archive suppressions
         uses: actions/upload-artifact@v3
         with:
           name: suppressions
           path: test/fuzz/**/suppressions
-          retention-days: 1
+          retention-days: 3
```

Comment:
I don't know that there's any benefit in retaining these results, unless we have a plan to use them

Reply:
What was the reasoning in having this in the first place?

Reply:
I'm not sure. As far as I know the fuzz tests are basically useless, (e.g. they've never caught a bug in the last 18 months, as far as I know, aside from sometimes unexpectedly breaking compile, in silly but legitimate ways.)